### PR TITLE
Twt dialog tkn

### PR DESCRIPTION
**Pull request by ajayparida**

### CODE REVIEW

**Review by krish2718 — COMMENTED** [review]
This should be split into two commits:
* Send dialog token to RPU
* Send TWT wake interval as is

File: drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fw_B/host_rpu_umac_if.h [hunk 1]
```diff
@@ -2868,6 +2869,7 @@ struct nrf_wifi_umac_event_new_scan_display_results {
 #define NRF_WIFI_EVENT_MLME_RX_SIGNAL_DBM_VALID (1 << 4)
 #define NRF_WIFI_EVENT_MLME_WME_UAPSD_QUEUES_VALID (1 << 5)
 #define NRF_WIFI_EVENT_MLME_RXMGMT_FLAGS_VALID (1 << 6)
+#define NRF_WIFI_EVENT_MLME_IE_VALID   (1 << 7)
```
Comment (krish2718) [inline drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fw_B/host_rpu_umac_if.h:2872]: This looks unrelated?

File: drivers/wifi/nrf700x/zephyr/src/zephyr_twt.c [hunk 2]
```diff
@@ -66,6 +66,7 @@ int wifi_nrf_set_twt(const struct device *dev,
 		/* Ceiling and conversion to milli seconds */
 		twt_info.twt_target_wake_interval_mantissa = ceil(mantissa * 1000);
 		twt_info.twt_target_wake_interval_exponent = exponent;
+		twt_info.dialog_token = twt_params->dialog_token;
```
Comment (krish2718) [inline drivers/wifi/nrf700x/zephyr/src/zephyr_twt.c:69]: Adding to interface file and populating can be squashed into a single commit.

**Review by sachinthegreen — APPROVED** [review]

**Review by rado17 — APPROVED** [review]

**Review by krish2718 — APPROVED** [review]

**Review by sr1dh48r — APPROVED** [review]
